CRM/Mailing/Event/BAO/Reply.php

```diff
@@ -194,23 +194,17 @@ public static function send($queue_id, &$mailing, &$bodyTxt, $replyto, &$bodyHTM
    * @param string $replyto
    *   Optional reply-to from the reply.
    */
-  private static function autoRespond(&$mailing, $queue_id, $replyto) {
-    $config = CRM_Core_Config::singleton();
-
-    $contacts = CRM_Contact_DAO_Contact::getTableName();
-    $email = CRM_Core_DAO_Email::getTableName();
-    $queue = CRM_Mailing_Event_DAO_Queue::getTableName();
-
-    $eq = new CRM_Core_DAO();
-    $eq->query(
-      "SELECT     $contacts.preferred_mail_format as format,
-                  $email.email as email,
-                  $queue.job_id as job_id,
-                  $queue.hash as hash
-        FROM        $contacts
-        INNER JOIN  $queue ON $queue.contact_id = $contacts.id
-        INNER JOIN  $email ON $queue.email_id = $email.id
-        WHERE       $queue.id = " . CRM_Utils_Type::escape($queue_id, 'Integer')
+  private static function autoRespond(&$mailing, $queue_id, $replyto): void {
+
+    $eq = CRM_Core_DAO::executeQuery(
+      'SELECT civicrm_email.email as email,
+               civicrm_mailing_event_queue.job_id as job_id,
+               civicrm_mailing_event_queuee.hash as hash
```

Comment:
@eileenmcnaughton missing a comma at the end of this one and also a typo in the table name

Reply:
thanks - will try to get to back to this before the rc is cut - otherwise will revert https://github.com/civicrm/civicrm-core/pull/24235 or do a temp patch on that